### ISSUE TITLE
Callouts should not be centre-justified

### ISSUE THREAD

**Issue by wbamberg** [mdn/yari#5511]
...it makes them harder to read.

For example: https://pr13476.content.dev.mdn.mozit.cloud/en-US/docs/Learn/JavaScript#modules

<img width="823" alt="Screen Shot 2022-03-07 at 9 39 27 AM" src="https://user-images.githubusercontent.com/432915/157088021-a56e97dd-7769-4c09-81f9-fcd9ea3db737.png">

**Comment by e3dio** [comment]
I like the text-align: center, you did not include screenshot of your proposed solution and its effect on other callouts

**Comment by wbamberg** [comment]
The proposed solution would be to have them left-justified, as we do for notes and warnings and, well, almost everything else.

Callouts aren't used very widely in MDN, really only in the Learn area and in the CSS layout cookbook. In most of these cases they're only a single line of text and look OK. But there's no particular reason a callout should have to be just one or two lines: they're really just like notes except they don't have to start with **Note:**, as in the case above, where I wanted it to start with a heading-like thing.

I don't think it's controversial to say that it's harder to read paragraphs of text when they are centre-justified, is it?

**Comment by e3dio** [comment]
I did not say it was controversial, I tried multiple different ways to display it and I liked center best, I think a couple screenshots of different callouts with your preferred solution would help

**Comment by caugner** [comment]
### 🍎 `text-align: center` (current)

![image](https://user-images.githubusercontent.com/495429/157491052-82615436-2335-4657-a6c4-a24955cb2f1d.png)

### 🍇  `text-align: justified`

![image](https://user-images.githubusercontent.com/495429/157490843-eadd6db5-61de-4968-9ad8-e3a80f0e40a4.png)

### 🍒 `text-align: left`

![image](https://user-images.githubusercontent.com/495429/157490950-a4c5954c-b18f-48d0-8890-7393c2df9e9d.png)

**Comment by e3dio** [comment]
But I don't know, kind of a toss up to me justify vs center

![image](https://user-images.githubusercontent.com/85405955/157494794-477e5608-5200-44a8-8edf-d5c88fa9c7a7.png)

**Comment by wbamberg** [comment]
Here are a couple of reasonably plausible references on why left-aligned is better for readability, and centering (in particular) is bad:

https://accessibility.huit.harvard.edu/design-readability
https://uxmovement.com/content/why-you-should-never-center-align-paragraph-text/

**Comment by e3dio** [comment]
Yes but this is a Callout, the point is to stand out from the rest of the page which is all left-aligned. You would never center a large amount of text, only special cases. Only my humble opinion of course

**Comment by wbamberg** [comment]
The example in this bug is exactly a "large amount of text", or at least large enough for centering to be inappropriate. In the specification for callouts there is no suggestion that they could not include multiple paragraphs of text, and in fact it says that they can: https://developer.mozilla.org/en-US/docs/MDN/Contribute/Markdown_in_MDN#notes_warnings_and_callouts . They are seen as the same sort of thing as notes and warnings, which are left-aligned.

**Comment by e3dio** [comment]
When I look at the page with Callouts, the #1 thing that draws my eyes to the Callout and makes it interesting and unique is the `text-align: center`. It's one of the few or only places on the site that is center. If you take that away it does not look interesting, just another text block. Is it really that difficult to read that it is causing issues?

**Comment by wbamberg** [comment]
I don't think I have anything useful to add to the issue at this point, so I'm out.

**Comment by caugner** [comment]
@e3dio Based on your behavior here and in other issues before, I would like to kindly remind you of our [Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/), and ask you to make sure that your behavior is in accordance with those guidelines at all times. 🙏 Thanks! 

@wbamberg I fully agree that those callouts shouldn't be centered, so just to clear: Do you prefer `text-align: left` or `text-align: justified`? You wrote "left-justified" further above, so I wasn't sure.

**Comment by wbamberg** [comment]
> @wbamberg I fully agree that those callouts shouldn't be centered, so just to clear: Do you prefer `text-align: left` or `text-align: justified`? You wrote "left-justified" further above, so I wasn't sure.

Yes, sorry. I meant `text-align: left` but `text-align: justified` would probably be OK too?

It might be worth thinking about whether callouts and notes and warnings should all the the same kind of alignment. I think of callouts as being like notes and warnings, except they don't have a predefined "heading" (i.e. **Note:** or **Warning:**), so you can supply your own. 

So in the case above I wanted to use a callout, because:

* this content felt like it wasn't part of the main flow of the page, and
* I wanted the heading "Our policy on modern JavaScript", not something "Note: Our policy on modern JavaScript", which would have read weird.

Does that make sense?

**Comment by caugner** [comment]
@wbamberg Makes sense to me! Since notes and warnings currently have `text-align: left`, I think that would make most sense for callouts as well, to be consistent.

Note that smaller callouts like [here](https://developer.mozilla.org/en-US/docs/Web/CSS#key_resources) would look as follows:

![image](https://user-images.githubusercontent.com/495429/157690149-3f8baaf9-ffe4-4fa6-819a-a91e9520721d.png)

Instead of:

![image](https://user-images.githubusercontent.com/495429/157690072-05f66595-918b-4a00-8568-9ca018877002.png)

Either that's okay, or we need to provide an optional CSS class for short callouts to be centered. 🤔

**Comment by wbamberg** [comment]
Thanks @caugner ! To me left-aligned looks totally fine here (but I am no designer).

**Comment by caugner** [comment]
@schalkneethling Any objection against making those callouts `text-align: left`?

**Comment by schalkneethling** [comment]
> @schalkneethling Any objection against making those callouts `text-align: left`?

Nope, makes sense to me.

**Comment by OnkarRuikar** [comment]
If we have concerned about callouts not standing out. Then we can increase shadow. Currently set shadows are almost non existent.  Light theme has 0.05 alpha and Dark has 0.2 alpha!

If we set shadow `0px 0px 1rem #2b2a33` then callouts standout like this:
1. small
![small](https://i.imgur.com/0LALBwh.png)

2. big
![big](https://i.imgur.com/wwPbH4y.png)

**Comment by schalkneethling** [comment]
Thank you for the suggestion @OnkarRuikar. I feel like that will be too distracting for readers. The focus should be the content. I believe the problem we aim to address here is that text centred like in examples are just hard to read for most people.
